channels/pl.m3u

```diff
@@ -1,16 +1,68 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="4FUNTV.pl" tvg-name="4FUN TV" tvg-country="PL" tvg-language="" tvg-logo="" group-title="",4FUN TV [Not 24/7]
 https://stream.4fun.tv:8888/hls/4f_high/index.m3u8
-#EXTINF:-1 tvg-id="4FUNDance.pl" tvg-name="4FUN Dance" tvg-country="PL" tvg-language="" tvg-logo="" group-title="",4FUN Dance
+#EXTINF:-1 tvg-id="4FUNDance.pl" tvg-name="4FUN Dance" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Music",4FUN Dance
 http://91.218.60.3:8001/play/a008
-#EXTINF:-1 tvg-id="4FUNKids.pl" tvg-name="4FUN Kids" tvg-country="PL" tvg-language="" tvg-logo="" group-title="",4FUN Kids
+#EXTINF:-1 tvg-id="4FUNKids.pl" tvg-name="4FUN Kids" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Music",4FUN Kids
 http://91.218.60.3:8001/play/a009
+#EXTINF:-1 tvg-id="AXNBlack.pl" tvg-name="AXN Black" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN Black (Poland)
+http://91.218.60.3:8001/play/a00h
+#EXTINF:-1 tvg-id="AXN.pl" tvg-name="AXN" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN (Poland) (1080p)
+http://91.218.60.3:8001/play/a00i
+#EXTINF:-1 tvg-id="AXN.pl" tvg-name="AXN" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN (Poland) (576p)
+http://91.218.60.3:8001/play/a00j
+#EXTINF:-1 tvg-id="AXNSpin.pl" tvg-name="AXN Spin" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN Spin (Poland) (1080p)
+http://91.218.60.3:8001/play/a00k
+#EXTINF:-1 tvg-id="AXNSpin.pl" tvg-name="AXN Spin" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN Spin (Poland) (576p)
+http://91.218.60.3:8001/play/a00l
+#EXTINF:-1 tvg-id="AXNWhite.pl" tvg-name="AXN White" tvg-country="PL" tvg-language="" tvg-logo="" group-title="Entertainment",AXN White (Poland) (576p)
+http://91.218.60.3:8001/play/a00m
```

Comment:
AXN is US channel: https://www.lyngsat.com/tvchannels/us/AXN-Polska.html

Reply:
Well AXN has got different divisions. While it is owned by Sony, the Polish versions is operated by AXN Europe, hence the fact that I didn't really know where to put it.